src-gui/src/renderer/components/modal/introduction/slides/Slide05_KeepAnEyeOnYourSwaps.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link, Typography } from '@material-ui/core'
+import SlideTemplate from './SlideTemplate'
+import imagePath from '../../../../../assets/mockHistoryPage.webp'
+
+export default function Slide05_KeepAnEyeOnYourSwaps(props: slideProps) {
+    return (
+        <SlideTemplate
+            title="Monitor Your Swaps"
+            stepLabel="Step 3"
+            {...props}
+            imagePath={imagePath}
+        >
+            <Typography>
+                While the Atomic Swap Protocol is secure, you should monitor active swaps in the history tab to ensure everything proceeds smoothly.
```

Comment:
I feel that we should inform the users a bit more specifically about the refund behaviour, something like:

"If the swap does not succeed after ~12 hours (72 blocks) you have a window of ~24 hours (144 blocks) to refund the Bitcoin. The app does this automatically. You just need to have it running at some point during the refund period."